contrib/completion/zsh/_docker-compose

```diff
@@ -20,7 +20,7 @@
 # -------------------------------------------------------------------------
 
 # Extracts all service names from docker-compose.yml.
-___docker-compose_all_services_in_compose_file() {
```

Comment:
There is normally a meaning about triple underscore prefix which is sub-functions used by complete function. Ex: `__docker-compose_services_from_build` using `___docker-compose_services_with_key`. But I'm okay about using only double underscore prefix.